scene/gui/tab_container.cpp

```diff
@@ -567,6 +567,8 @@ void TabContainer::remove_child_notify(Node *p_child) {
 	// TabBar won't emit the "tab_changed" signal when not inside the tree.
 	if (!is_inside_tree()) {
 		call_deferred("_repaint");
+	} else if (tab_bar->get_tab_count() > 0 && is_current_tab) {
+		tab_bar->call_deferred("emit_signal", SNAME("tab_changed"), tab_bar->get_current_tab());
```

Comment:
```suggestion
		tab_bar->call_deferred(SNAME("emit_signal"), SNAME("tab_changed"), tab_bar->get_current_tab());
```
Although you could avoid method string by using `callable_mp()` (it's cumbersome though, until #67730 gets merged).